lib/oat/adapters/siren.rb

```diff
@@ -94,7 +94,7 @@ def initialize(name)
             @data = { :name => name }
           end
 
-          %w(type value title).each do |attribute|
+          %w(category required class type value title).each do |attribute|
```

Comment:
Not alphabetic, but still fine :+1:

Reply:
wasn't alphabetical before either...

Reply:
Because of this I accepted the additions :grin: